### PR TITLE
Phs 87 community modal design

### PR DESCRIPTION
**Pull request by Prashishgt**
waiting for code review

### CODE REVIEW

File: src/Layouts/Community/CommunityGrid.tsx [hunk 1]
```diff
@@ -1,48 +1,66 @@
...
+            />
+          </button>
+        ))}
       </div>
-      <div className="grid__post__item">
-        <img
-          className="community__grid__item__image"
-          src="https://scontent.cdninstagram.com/v/t51.29350-15/352642049_655550593278091_2676964400428628937_n.jpg?_nc_cat=105&ccb=1-7&_nc_sid=8ae9d6&_nc_ohc=djBfs2IBDKwAX9gLd1O&_nc_ht=scontent.cdninstagram.com&edm=AM6HXa8EAAAA&oh=00_AfAJ2vuFDL5B8Y3QF2GhdSZt7537lNrFJQga2sD4nJ6zMQ&oe=648F7795"
-          alt=""
+      {selectedImageIndex !== null && (
```
Comment (sarojghalan) [inline src/Layouts/Community/CommunityGrid.tsx:54]: for better check with selectedImageIndex with !== [].

**Review by sarojghalan — COMMENTED** [review]
please complete the following comment

File: src/assets/CommunityModalIcons/index.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import CloseIcon from '@mui/icons-material/Close';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import EmailIcon from '@mui/icons-material/Email';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import PinterestIcon from '@mui/icons-material/Pinterest';
+import InstagramIcon from '@mui/icons-material/Instagram';
```
Comment (subhashishshah) [inline src/assets/CommunityModalIcons/index.ts:6]: folder structure improvements

File: src/assets/dummy/index.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import img1 from './1.jpg';
```
Comment (subhashishshah) [inline src/assets/dummy/index.ts:1]: name should always be meaningful

**Review by subhashishshah — APPROVED** [review]

**Pull request merged**